### ISSUE TITLE
Tweets starting with "RT:" are not detected as retweets

### ISSUE THREAD

**Issue by trivikr** [marcomaroni-github/twitter-to-bluesky#52]
## Describe the bug
Tweets starting with "RT:" are not detected as retweets

## Steps to reproduce

I'm not sure how I used to retweet in 2009 before retweeting was standardized, but half of my retweets start with `RT:`
Example https://x.com/trivikram/status/1429604586

#### Observed behavior
These retweets are not detected as retweets my this migration tool, and a bluesky skeet is created.

#### Expected behavior
Tweets starting with `RT:` to be detected as retweets?

## Additional context
I just ended up deleting those posts once they were created on BlueSky.

**Comment by clemensv** [comment]
If there were command line options, "retweet prefixes" is an option I'd add, @marcomaroni-github  ;)

**Comment by marcomaroni-github** [comment]
@trivikr the tweet you posted as an example is not detected as RT because it has a colon after RT

**Comment by marcomaroni-github** [comment]
@trivikr The colon after RT did not make a tweet an RT, there should have been a space after RT, even the example you gave does not seem to me to be identified as an RT

**Comment by clemensv** [comment]
In the earliest days, "RT" was just a convention and not a feature

**Comment by lucidillusions** [comment]
yeah, people used to quote a post and add RT to  show it was the person's tweet and then add their take to it.

**Comment by trivikr** [comment]
I also used to add suffix sometimes `(via @alias)`. I think it was through Echofon.

**Comment by trivikr** [comment]
To unblock myself, I just did a deep dive in `data/tweets.js` in my archive, and wrote a script to explicitly add `RT ` prefix to the `full_text` of these tweets. The module than treated these tweets as retweets.
